packages/myst-frontmatter/src/page/page.yml

```diff
@@ -40,7 +40,7 @@ cases:
         a: b
       subtitle: sub
       short_title: short
-      date: 14 Dec 2021
+      date: '14 Dec 2021'
```

Comment:
These quotes don't change how the contents are parsed, but it makes it clear we are ingesting strings not YAML timestamps.